doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -828,6 +828,10 @@ Wi-Fi samples that used :ref:`zephyr:bluetooth-hci-ipc-sample` or :ref:`zephyr:n
 
   * Modified to use the :ref:`lib_wifi_ready` library to manage the Wi-Fi use.
 
+* :ref:`wifi_softap_sample` sample:
+
+  * Modified to use the :ref:`lib_wifi_ready` library to manage the Wi-Fi use.
+
```

Comment:
Please add a comment to add this entry in https://github.com/nrfconnect/sdk-nrf/pull/15961 PR and remove it from here.

Reply:
Added.